src/identity/email-password-identity-provider/tempAccount.ts

```diff
@@ -0,0 +1,73 @@
+import type { KoaContextWithOIDC } from 'oidc-provider';
+import { v4 } from 'uuid';
+
+const store = new Map();
```

Comment:
What is in the store? Can we have a better name?
Why is the store shared across all instances?

Reply:
Yep, this is all temp stuff. It will be refactored.